### PR TITLE
[#2637] [PORT] Testing naughty strings against TextPrompt

### LINKED ISSUE #2637: [PORT] Testing naughty strings against TextPrompt
> Port this change from botbuilder-dotnet/master branch:
https://github.com/microsoft/botbuilder-dotnet/pull/4364

Fixes #3976 

# Changed projects
* Microsoft.Bot.Builder
* Microsoft.Bot.Builder.Dialogs.Tests

### PR DESCRIPTION
**Pull request by denscollo**
Fixes #2637

## Description

This pull request adds a test in bot builder dialogs to test naughty strings against TextPrompt. This implementation is ported from the PR 4364 from BotBuilder-DotNet.

## Specific Changes

- Added it `should call TextPrompt with naughty strings`.
- Added file `naughtyStrings.txt` in the resources folder.
- Added `line-reader` reference to read the lines from the naughty strings file.
- Minor Eslint fixes.

## Testing
Below you can see the added test running.
image

### CODE REVIEW

File: libraries/botbuilder-dialogs/package.json [hunk 1]
```diff
@@ -26,7 +26,8 @@
     "@microsoft/recognizers-text-suite": "1.1.4",
     "@types/node": "^10.17.27",
     "botbuilder-core": "4.1.6",
-    "globalize": "^1.4.2"
+    "globalize": "^1.4.2",
+    "line-reader": "^0.4.0"
```
Comment (joshgummersall) [inline libraries/botbuilder-dialogs/package.json:30]: This should be a dev dependency as it’s only used for testing.

**Review by joshgummersall — CHANGES_REQUESTED** [review]
One quick fix then it’s good to go!

File: libraries/botbuilder-dialogs/package.json (hunk 1 above)
Comment (denscollo) [reply libraries/botbuilder-dialogs/package.json]: Hi @joshgummersall we just pushed the changes. Thanks!

**Review by joshgummersall — COMMENTED** [review]
One more quick fix.

**Review by joshgummersall — APPROVED** [review]

**Pull request merged**